packaging/mingw-w64/configure-build-msys2-mingw.sh

```diff
@@ -10,15 +10,48 @@ cmake \
     -D RDKAFKA_BUILD_TESTS=ON \
     -D WITH_LIBDL=OFF \
     -D WITH_PLUGINS=OFF \
-    -D WITH_SASL=ON \
```

Comment:
Suggest creating a new file, configure-build-msys2-mingw-static.sh, since this file is supposed to build a dynamic lib.

Reply:
Done